.github/workflows/changes.yml

```diff
@@ -188,7 +188,7 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ inputs.int_tests }}
     outputs:
-      all-int: ${{ steps.filter.outputs.all-int}}
+      all-int: ${{ steps.filter.outputs.all-tests}}
```

Comment:
What was this change for?

Reply:
The source code in ci_paths.rs changed to be more generic since that is shared by both the int tests and e2e tests subcommand.

Reply:
Ah I see. I think my confusion stems from the fact that I expected to see `all-int` and `all-e2e` as separate outputs. Here you are saying that its one output that encompasses both.

Should the `all-int` key be updated to be `all-tests` then too?

Reply:
I just pushed up https://github.com/vectordotdev/vector/pull/19666/commits/0c6a4c481a249e7fb7aea50915b86ad54a358ea3 to adjust the keys.

IMO its a complete 50/50 toss up on whether it makes more sense to do it this way, or to have the vdev output produce the stdout `all-e2e` and `all-int` , since that would entail adding another param to the ci_paths function 🤷  . While I feel the `all-tests` is generic and doesn't really reduce the clarity of the output from the ci paths subcommand.

Reply:
Yeah, I think `all-tests` may be confusing without more context since one might expect it to include unit tests and other sorts of tests too on top of the e2e and integration tests.

Reply:
I don't feel strongly no 😄

Reply:
😃  these are the things that matter! lol